examples/convert_to_aster.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
+#
+# Copyright (C) 2018 COAL Developers
+#
+# This program is free software; you can redistribute it and/or 
+# modify it under the terms of the GNU General Public License 
+# as published by the Free Software Foundation; version 2.
+#
+# This program is distributed in the hope that it will be useful, 
+# but WITHOUT ANY WARRANTY; without even the implied warranty 
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public 
+# License along with this program; if not, write to the Free 
+# Software Foundation, Inc., 51 Franklin Street, Fifth 
+# Floor, Boston, MA 02110-1301, USA.
+# encoding: utf-8
+
+'''
+Spectral Version 7 .txt file converted to ASTER .txt file format
+
+@author:     COAL Developers
+
+@copyright:  2018 COAL Developers. All rights reserved.
+
+@license:    GNU General Public License version 2
+
+@contact:    coal-capstone@googlegroups.com
+'''
+
+from argparse import ArgumentParser
+from argparse import RawDescriptionHelpFormatter
+import sys
+import os
+from sys import path
+from os import getcwd
+import inspect
+import math
+import numpy
+
+#This will convert a Spectral Version .txt file to a ASTER .txt file
+library_filename = 'splib07a_Alizarin_crimson_(dk)_GDS780_ASDFRa_AREF.txt'
+with open(library_filename,'r') as f:
+    for i, l in enumerate(f):
+        pass
+
+f = open(library_filename,'r')
+nameTemp = f.readline()
+name = nameTemp[23:]
+k = 0
+x = open('2.txt','w')
```

Comment:
```2.txt``` is a ridiculous name for any file however seeing as it is a _temporary_ file AND you clean it up I suppose this is OK.